build-tools/xaprepare/xaprepare/ConfigAndData/BuildAndroidPlatforms.cs

```diff
@@ -5,7 +5,8 @@ namespace Xamarin.Android.Prepare
 {
 	class BuildAndroidPlatforms
 	{
-		public const string AndroidNdkVersion = "19c";
+		public const string AndroidNdkVersion = "20";
+		public const string AndroidNdkPkgRevision = "20.0.5594570";
```

Comment:
Why a field and not hardcoded in the `AndroidToolchainComponent` creation, below?  `AndroidNdkPkgRevision` doesn't appear to be used anywhere else.

Reply:
It's to keep the two values together here - or otherwise it might be easy to forget to update one or another.